.github/workflows/cypress-integration-test.yml

```diff
@@ -3,110 +3,121 @@ name: Cypress Integration Test Workflow
             yarn_cache_folder: .cache/yarn
             path: |
               .cache/yarn
               node_modules
-  
-        - name: Configure AWS Credentials
-          uses: ./.github/workflows/configure-aws-credentials
-          with:
-            aws_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-            aws_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-            aws_region: us-gov-west-1
-  
-        - name: Get Mapbox Token
-          uses: ./.github/workflows/inject-secrets
-          with:
-            ssm_parameter: /dsva-vagov/vets-website/dev/mapbox_token
-            env_variable_name: MAPBOX_TOKEN
-  
-        - name: Build
-          run: yarn build --verbose --buildtype=${{ env.BUILDTYPE }}
-          timeout-minutes: 30
-  
-        - name: Generate build details
+
+        - name: Install server dependencies
+          working-directory: vets-website
           run: |
-            cat > build/${{ env.BUILDTYPE }}/BUILD.txt << EOF
-            BUILDTYPE=${{ env.BUILDTYPE }}
-            NODE_ENV=production
-            BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}")
-            CHANGE_TARGET=null
-            RUN_ID=${{ github.run_id }}
-            RUN_NUMBER=${{ github.run_number }}
-            REF=${{ github.sha }}
-            BUILDTIME=$(date +%s)
-            EOF
-  
-        - name: Compress and archive build
-          run: tar -C build/${{ env.BUILDTYPE }} -cjf ${{ env.BUILDTYPE }}.tar.bz2 .
-  
-        - name: Upload build artifact
-          uses: ./.github/workflows/upload-artifact
-          with:
-            name: ${{ env.BUILDTYPE }}.tar.bz2
-            path: ${{ env.BUILDTYPE }}.tar.bz2
-            retention-days: 1
-  
-    call-integration-test-workflow:
-      name: Call Integration Test Workflow
-      uses: department-of-veterans-affairs/vagov-cypress-integration-tests/.github/workflows/cypress-test-integration-workflow.yml@main
-      secrets: inherit
-  
-    notify-failure:
-      name: Notify Failure
-      runs-on: ubuntu-latest
-      if: ${{ failure() || cancelled() }}
-      needs: call-integration-test-workflow
+            yarn install
 
-      steps:
-      - name: Checkout
-        uses: actions/checkout@cd7d8d697e10461458bc61a30d094dc601a8b017
+        - name: Start server
+          run: |
+           mkdir build
+           node src/platform/testing/e2e/test-server.js --buildtype=../vets-website --port=3001 &
 
-      - name: Notify Slack
-        uses: ./.github/workflows/slack-notify
-        continue-on-error: true
-        with:
-          payload: '{"attachments": [{"color": "#FF0800","blocks": [{"type": "section","text": {"type": "mrkdwn","text": "vets-website manual dev/staging deploy failed!: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}>"}}]}]}'
-          channel_id: ${{ env.VETS_WEBSITE_CHANNEL_ID }}
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Slack integration was removed from the MVP, and will be added as a feature, pending Platform adoption.